instana/collector/base.py

```diff
@@ -151,15 +151,12 @@ def prepare_and_report_data(self):
         @return: Boolean
         """
         if env_is_test is False:
```

Comment:
Since we are refactoring, I wolud refactor this statement to an early return so the rest of the logic (beginning with a `with` statement) is not behind a condition.